test/cmd/wait.sh

```diff
@@ -94,6 +94,21 @@ EOF
     # Clean deployment
     kubectl delete deployment dtest
 
+    # create test data
+    kubectl create deployment test-3 --image=busybox
+
+    # wait with jsonpath without value to succeed
+    set +o errexit
```

Comment:
I don't think we'd need this, since we are using `2>&1`

Reply:
It is not entirely correct. The command `set +o errexit` "tells" the script to not terminate if one of the following executions end with non-zero return code. The `2>&1` merely redirects all prints to `stderr` to be in `stdout`.
If you will look on other lines in this script or into other scripts, you will see that they all follow similar pattern.

Reply:
You are right. Especially considering that the few lines above was added by me with the same :)